src/components/dev-hub/podcast-player/player-contols.js

```diff
@@ -0,0 +1,98 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+import Button from '~components/dev-hub/button';
+import FastForwardIcon from '~components/dev-hub/icons/audio-player/fast-forward-icon';
+import PauseIcon from '~components/dev-hub/icons/audio-player/pause-icon';
+import PlayIcon from '~components/dev-hub/icons/audio-player/play-icon';
+import RewindIcon from '~components/dev-hub/icons/audio-player/rewind-icon';
+
+import { size } from '~components/dev-hub/theme';
+
+const buttonStyles = css`
+    margin-right: 12px;
+    padding: ${size.tiny};
+    &:active {
+        filter: brightness(60%);
+    }
+`;
+
+export const ControlsContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    justify-content: center;
+`;
+
+const StyledTimeControlButton = styled(Button)`
+    ${buttonStyles},
+    &:hover {
+        svg {
+            transform: scale(1.15);
+        }
+    }
+`;
+
+const StyledMainButton = styled(Button)`
+    ${buttonStyles},
+    &:hover {
+        svg {
+            circle {
+                stroke-width: 4;
+                r: 38;
+            }
+            path {
+                stroke-width: 4;
+            }
+        }
```

Comment:
This might be good to extract into something that uses the `css` function like above for `buttonStyles`.